src/control/lib/control/mocks.go

```diff
@@ -540,3 +545,101 @@ func MockFormatResp(t *testing.T, mfc MockFormatConf) *StorageFormatResp {
 		HostStorage: hsm,
 	}
 }
+
+type (
+	MockStorageConfig struct {
+		TotalBytes uint64
+		AvailBytes uint64
+	}
+
+	MockScmConfig struct {
+		MockStorageConfig
+	}
+
+	MockNvmeConfig struct {
+		MockStorageConfig
+		Rank system.Rank
+	}
+)
+
+func MockStorageScanResp(testRunner *testing.T,
```

Comment:
I imagine this is probably duplicated somewhere else in the package

Reply:
I have already searched for something like this, but I was sadly not able to find it. I will double check it as I am starting to be a little bit more familiar with the code.

- [ ] Look for duplicated code
- [ ] Remove duplicated code if needed

Reply:
There is indeed a `MockServerScanResp` which is also mocking a `ctlpb.StorageScanResp`. However, I choosed to not use it: using it will imply some heavy modifications (and a noticeable increase of its complexity) of this mocker and a very small portions of code could be reused. Thus, I priviligied to duplicate a small portion of code and keep the thing simple: [KISS ](https://fr.wikipedia.org/wiki/Principe_KISS) ;-)